temporian/core/operators/simple_moving_average.py

```diff
@@ -29,12 +29,12 @@ class SimpleMovingAverage(Operator):
   def __init__(
       self,
       data: Event,
-      window_length: int,
+      window_length: str,
       sampling: Optional[Event] = None,
   ):
     super().__init__()
 
-    self.window_length = window_length
+    self.add_attribute("window_length", window_length)
```

Comment:
For later: It would be good to parse the window_length before the graph execution as to catch early any issues.